livesql/binlog.go

```diff
@@ -71,18 +71,18 @@ func getPosition(conn *sql.DB) (mysql.Position, error) {
 //
 // NewBinlog verifies that the given DB has been correctly configured for
 // streaming changes.
-func NewBinlog(ldb *LiveDB, host string, port uint16, username, password, database string) (*Binlog, error) {
+func NewBinlog(ldb *LiveDB, binlogDB *sql.DB, host string, port uint16, username, password, database string) (*Binlog, error) {
```

Comment:
should we add a new `NewBinlogWithSource` or similar so that this isn't a breaking change?

Reply:
sg

Reply:
done